mindocr/utils/callbacks.py

```diff
@@ -208,16 +208,15 @@ def __init__(self,
         self.step_start_time = time.time()
 
         self._losses = []
-        if device_num is not None:
-            reduce_sum = ms.ops.AllReduce()
 
-            @jit
-            def reduce(x):                          # lamda expression is not supported in MindSpore
-                return reduce_sum(x) / device_num   # average loss across all cards
+        self._reduce_sum = ms.ops.AllReduce()
+        self._device_num = device_num
+        # lamda expression is not supported in jit
+        self._loss_reduce = self._reduce if device_num is not None else lambda x: x
 
-            self._loss_reduce = reduce
-        else:
-            self._loss_reduce = lambda x: x
+    @jit
+    def _reduce(self, x):
```

Comment:
@zhtmike @SamitHuang Please check.

Reply:
Not sure whether running with ms_function in callback is a stable choice. For MS 1.9, pynative with ms_function is not as stable as MS 2.0. If using ms_function is risky, i don't it is worthy to add jit/ms_function considering the ignorable acceleration on this  one-step division computation.